armi/physics/fuelPerformance/plugin.py

```diff
@@ -55,50 +51,3 @@ def defineParameters():
         from armi.physics.fuelPerformance import parameters
 
         return parameters.getFuelPerformanceParameterDefinitions()
-
-    @staticmethod
-    @plugins.HOOKIMPL
-    def afterConstructionOfAssemblies(assemblies, cs):
```

Comment:
**Question**: Have you check other downstream ARMI plugins to make sure they don't depend on `afterConstructionOfAssemblies()`?

There are some common ones you could `git grep` through to see if this PR will break a bunch of plugins, you know?

Reply:
But the method is empty after the removal of `_setBOLBond`. And yes, we are certain that `_setBOLBond` is not used in any downstream plugins, with the exception of our internal metal fuel performance plugin, to which the `_setBOLBond` call has been added. 

(Let me know if you want a link to the relevant PR in that plugin, and I can send it to you offline.)

And then once `afterConstructionOfAssemblies` is empty, it is the same as that in its `super` class, so removing it shouldn't be a problem.

But let me know if you think otherwise.